Feature/update shell task (#1451)
PrefectHQ/prefect: CHANGELOG.md

- Rename `CloudEnvironment` to `DaskKubernetesEnvironment` - [#1250](https://github.com/PrefectHQ/prefect/issues/1250)
- Remove unused `queue` method from all executors - [#1434](https://github.com/PrefectHQ/prefect/pull/1434)

> - Updating ShellTask to work more like Airflow Bash Operator for streaming logs and returning values, which changes behavior of task - [#1451](https://github.com/PrefectHQ/prefect/pull/1451)

Reviewer: ```suggestion
- Update `ShellTask` to return only the last line of stdout, as a string - [#1451](https://github.com/PrefectHQ/prefect/pull/1451)
```

just to be clear on what the breaking change is

Author: change made via push
